src/Symmetry/Mirror_Symmetry.jl

```diff
@@ -0,0 +1,35 @@
+struct MirrorSymmetry{T}
+    symmetry_plane::Plane{T}
+end
+
+function MirrorSymmetry{T}(origin::CylindricalPoint{T}, normal::CylindricalVector{T}) where {T}
+    _origin = CartesianPoint(origin)
+    _normal = CartesianVector(CartesianPoint(origin + normal)-CartesianPoint(origin))
+    MirrorSymmetry{T}(Plane{T}(_origin, _normal))
+end
+
+function MirrorSymmetry{T}(origin::CartesianPoint{T}, normal::CartesianVector{T}) where {T}
+    MirrorSymmetry{T}(Plane{T}(origin, normal))
+end
+
+function MirrorSymmetry(axis, value::T, units::NamedTuple = default_unit_tuple()) where {T<: SSDFloat}
+    length_unit = units.length
+    angle_unit = units.angle
+    if axis == "phi" || axis == "φ"
+        eps = 1e-7
+        MirrorSymmetry{T}(CylindricalPoint{T}(r=1, φ = to_internal_units(value * angle_unit)),
+            CylindricalVector{T}(0, to_internal_units(eps * length_unit), 0))
```

Comment:
Why this small value and not just `1`? 

Is this correct? A `Plane` is defined by a `CartesianPoint` and a `CartesianVector`.

If we find out that we get troubles with numerical conversions between Cartesian and cylindrical coordinates, we should define a dedicated struct like `PhiMirrorSymmetry`.

Reply:
This was a convenience solution for the time being, because there is no function converting between cylindrical and cartesian vector. For the plane we also have the origin, meaning we can define the transformation. I used the analytical expression now. Still it contains a sin and cos, leading to numerical errors in the range of the machine precision. Are we okay with that or should we round?